script/src/syscalls/load_code.rs

```diff
@@ -99,28 +99,25 @@ impl<'a, Mac: SupportMachine, DL: DataLoader> Syscalls<Mac> for LoadCode<'a, DL>
             return Ok(true);
         }
         let cell = cell.unwrap();
-        let output = self.data_loader.lazy_load_cell_output(&cell);
+        let output = &cell.cell_output;
 
-        if content_offset >= output.data.len()
-            || (content_offset + content_size) > output.data.len()
+        if content_offset >= output.data_bytes as usize
+            || (content_offset + content_size) > output.data_bytes as usize
             || content_size > memory_size
         {
             machine.set_register(A0, Mac::REG::from_u8(SLICE_OUT_OF_BOUND));
             return Ok(true);
         }
+        let data = self.data_loader.load_cell_data(cell).expect("cell data");
```

Comment:
i don't understand the usage of `output.data_bytes` here, can we move this line 111 to line 104 and use the data length directly? could you explain the `output.data_bytes` design detail?

Reply:
It's for avoiding load data from DB, when we only want to calculate the occupied_capacity of a CellOutput.